moveit_core/collision_detection/CMakeLists.txt

```diff
@@ -1,3 +1,7 @@
+project(moveit_collision_detection
+  VERSION 1.0.2
+  LANGUAGES CXX
+)
```

Comment:
I realize the problem. This is not what we want to do. We don't want to add more `project` calls. All of these need to be deleted. You only need to change _existing_ `project()` calls. Please don't add any more.

Reply:
Got it, I will revert back with the changes